source/includes/api/performance/_allocated_claims.md

```diff
@@ -0,0 +1,80 @@
+## Allocated Claims
+
+### Create Allocated Claims
+
+Endpoint to create allocated claims against specific promotion. Claims are created by using the `data_field` names from above as key/value pairs. Your request must include user group name or company name, depending on the promotion configuration.
```

Comment:
"user group name" -> "user group id"?
"company name" -> "company id"?

Reply:
Good spot, sorry this was pulled straight over from the ticket description.